pages/textbook-beta/summer-school/introduction-to-quantum-computing-and-quantum-hardware-2020.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <main class="summer-school-page">
+    <TextbookBetaContentMenuSection
+      class="summer-school-page__content-menu"
+    />
+    <AppPageHeaderWithImage :cta="startLearningCTA" :back-link="backToTextbookHomeLink">
+      <template slot="title">
+        {{ headerTitle }}
+      </template>
+      <template slot="description">
+        <p v-text="headerDescription" />
+      </template>
+      <template slot="image">
+        <img class="app-page-header-with-img__image" :src="headerImg">
+      </template>
+    </AppPageHeaderWithImage>
+    <PrerequisiteMaterialSection class="summer-school-page__section" :data="summerSchool2020Prerequisites" />
+    <CoursePagesSection class="summer-school-page__section" :courses="courses" :img-base="imageUrlBase" />
+  </main>
+</template>
+
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/textbookBetaContent'
+import { Component } from 'vue-property-decorator'
+import QiskitPage from '~/components/logic/QiskitPage.vue'
+import { GeneralLink, START_LEARNING_URL } from '~/constants/appLinks'
+
+@Component({
+  layout: 'default-max',
+  head () {
+    return {
+      title: '2020 Qiskit global summer school on quantum computing and quantum hardware'
+    }
+  }
+})
+export default class SummerSchoolCoursePage extends QiskitPage {
+  routeName: string = 'summer-school/intro-quantum-computing-quantum-hardware-2020'
+
+  headerTitle = '2020 Qiskit Global Summer School on Quantum Computing and Quantum Hardware'
+  headerDescription = 'This course is an introduction to the world of quantum computing, with an exploration of some of the key quantum algorithms and their implementations using quantum circuits, as well as the quantum hardware that is designed to run these algorithms. The course was first offered during the Qiskit Global Summer School in July 2020 as a two-week intensive summer school. There are 27 lectures in this course, which cover the material in 9 lecture notes and 9 associated labs.'
+
+  headerImg = '/images/textbook-beta/summer-school/intro-quantum-computing-quantum-hardware-2020/header.png'
+
+  backToTextbookHomeLink: GeneralLink = {
+    url: '/textbook-beta',
+    label: 'Home',
+    segment: {
+      cta: 'back-to-textbook-home', location: 'header'
+    }
+  }
+
+  startLearningCTA: GeneralLink = {
+    url: START_LEARNING_URL.SummerSchool2020,
+    label: 'Start learning',
+    segment: {
+      cta: 'introduction', location: 'header'
+    }
+  }
+
+  courses: Course[] = [
+    {
+      image: 'QGSS-Lec1-CoverImage.png',
+      label: 'Qubits and Quantum States, Quantum Circuits, Measurements',
+      segment: { cta: 'lecture-1', location: 'summer-school-2020' },
+      url: 'https://learn.qiskit.org/summer-school/2020/qubits-states-circuits-measurements'
```

Comment:
Small question here, regarding the URL structure. 

When looking at the preview branch, I'm seeing `/course/` in the URL.

For example: `https://platypus-pr-312.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/2020/qubits-states-circuits-measurements`

I also see that `https://platypus-pr-312.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/summer-school/2020/qubits-states-circuits-measurements` works just fine...  (here, there is no `course` in the URL)

I take it the summer-school URL is the right one though, yeah?

Reply:
yeah, `course/` is the default that gets generated for all pages by Mathigon, but we added `summer-school/` specifically for the summer school content so at the moment both work. 

however, it may be worth having a greater discussion if we want to block (404), redirect (from `course/` to `summer-school/`), or just leave summer school content as is